src/content/errors/6002.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "6002"
+slug: "6002"
+description: NG6002 - NgModule import errors
+author: Rafael Mestre
+authorTwitter: mestre_dev
+date: Aug 23, 2023
+---
+
+# NG6002
+
+`NG6002` is a compiler error that indicates problems with imports. It is aliased as `NGMODULE_INVALID_IMPORT` in the `ErrorCode`s enum.
```

Comment:
Looks like we might be missing the ' before the s

![image](https://github.com/ngCaribe/angular-errors/assets/62297014/46b9b1d6-3b23-44de-821b-a980503b4486)